### PR TITLE
ASB-300 add search box for representation

### PR DESCRIPTION
**Pull request by pk2453**

### CODE REVIEW

File: packages/forms-web-app/src/controllers/projects/representations.js [hunk 1]
```diff
@@ -18,6 +21,7 @@ exports.getRepresentations = async (req, res) => {
       representations: representationsResponse.data.representations,
       paginationData,
       pageOptions,
+      searchTerm,
```
Comment (cpcundill) [inline packages/forms-web-app/src/controllers/projects/representations.js:24]: Need a `queryUrl` containing the searchTerm to be passed down for use in the pagination bar.  See documents as a reference example.

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/controllers/projects/representations.js (hunk 1 above)
Comment (pk2453) [reply packages/forms-web-app/src/controllers/projects/representations.js:24]: It was already added, jest need to refresh as it was in 2nd commit

Comment (cpcundill) [reply packages/forms-web-app/src/controllers/projects/representations.js:28]: I think you're missing the corresponding `{{queryUrl}}` being appended to the URL used within the pagination bar, i.e. `pageLinkTemplate`?

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/controllers/projects/representations.js (hunk 1 above)
Comment (pk2453) [reply packages/forms-web-app/src/controllers/projects/representations.js:28]: added to pagination now

**Review by cpcundill — APPROVED** [review]

**Pull request merged**